package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Backports native emoji characters to browsers that don't support them by replacing the characters with fallback images.",
   "main": "dist/index.umd.js",
   "module": "dist/index.esm.js",
+  "types": "dist/index.d.ts",
```

Comment:
This key is optional if the file is at the root and named `index.d.ts`. I think we should include this key though, because it short circuits a complex [module resolution](https://www.typescriptlang.org/docs/handbook/module-resolution.html) search through the file system. Just telling the compiler where the file is could save some build time with many TypeScript dependencies.